### ISSUE TITLE
 ERROR -- : MailHandler: Validation failed: Start time cannot be blank

### ISSUE THREAD

**Issue by mohitmahajan79** [ThatCompany/that_meeting#4]
Seeing below issue while creating Redmine ticket via mail, I am using Meeting Plugin in all trackers and for meeting invite start date and time is mandatory.
`
ERROR -- : MailHandler: Validation failed: Start time cannot be blank`

I have added allow_override=all to the script, but it is still giving me same error

Below are the values I am passing via email

Project: ABC 
Tracker: Bug
 Priority: High 
Status: New 
Assigned to: mohit mahajan 
Start date: 2021-07-14 
Start time: 11:00 AM 
End time: 01:00 PM

In short I'm not able to pass start time keyword via mail

**Comment by s-andy** [comment]
This has been fixed in the develop branch.

Be aware, that the develop branch includes many new interesting features (available for Redmine 4+ only).
